Studio/Dockerfile.app

```diff
@@ -1,6 +1,6 @@
 # Base image with OS and dependencies
 # The base image does NOT include the Hackolade Studio application, which instead gets downloaded as part of the operations below
-FROM hackolade/studio:latest@sha256:3faf1ca123a731a39bf1f630968bd43e919b9b43775de6558aaa85ebeb8c2d50
```

Comment:
@lenchvolodymyr won't we have the same issue we faced with CICd images I had to downgrade back to 18.04 because of glibc backward compatibility?

Reply:
Also we should setup a regular build of these images to be published on HUB with os and lib security fixes